refactor: rename blockscout to atlascan across the project

r8bywork opened the pull request:
rename blockscout to atlascan across the project

r8bywork committed:
refactor: rename blockscout to atlascan across the project

r8bywork committed:
fix: jest tests

r8bywork committed:
fix: jest tests

r8bywork committed:
fix: playwright tests

r8bywork committed:
fix: playwright tests

ales-tsurko commented on deploy/services/logs/prod/error.log:
do we need this log to be tracked by git?

ales-tsurko commented on lib/metadata/__snapshots__/generate.test.ts.snap:
```diff
@@ -2,36 +2,36 @@
 exports[`generates correct metadata for: static route 1`] = `
 {
-  "description": "Blockscout is the #1 open-source blockchain explorer available today. 100+ chains and counting rely on Blockscout data availability, APIs, and ecosystem tools to support their networks.",
+  "description": "Atlascan is the #1 open-source blockchain explorer available today. 100+ chains and counting rely on Atlascan data availability, APIs, and ecosystem tools to support their networks.",
```
well, I'm not sure about this:) it's better to keep blockscout here

ales-tsurko commented on playwright/TestApp.tsx:
```diff
@@ -36,7 +36,7 @@ const defaultAppContext = {
 // >>> Web3 stuff
 const chains = [ mainnet ];
-const WALLET_CONNECT_PROJECT_ID = 'PROJECT_ID';
+const WALLET_CONNECT_PROJECT_ID = '1ca62e51c2fd3eab3fa030d624000507';
```
what this variable for? I saw you previously used this value. where does it come from? is it safe to expose?

ales-tsurko commented on ui/address/AddressCoinBalance.pw.tsx:
```diff
@@ -7,7 +7,7 @@ import buildApiUrl from 'playwright/utils/buildApiUrl';
 import AddressCoinBalance from './AddressCoinBalance';
 
-const addressHash = 'hash';
+const addressHash = '0x08e390762f64ABA6F9F9269589e1A702623e90F1';
```
could you please explain the value for this variable too?

ales-tsurko commented on ui/address/AddressDetails.pw.tsx:
```diff
@@ -46,8 +46,8 @@ test('contract +@mobile', async({ mount, page }) => {
   await expect(component).toHaveScreenshot({
-    mask: [ page.locator(configs.adsBannerSelector) ],
-    maskColor: configs.maskColor,
+    // mask: [ page.locator(configs.adsBannerSelector) ],
```
am I correct this test checks validity of the ad banner look? if so, please remove the whole blocks. we don't need commented code in our base.

ales-tsurko commented on ui/address/contract/ContractCode.pw.tsx:
```diff
@@ -27,7 +27,7 @@ const test = base.extend<socketServer.SocketServerFixture>({
 
 test('full view +@mobile +@dark-mode', async({ mount, page }) => {
-  await page.route('https://cdn.jsdelivr.net/npm/monaco-editor@0.33.0/**', (route) => route.abort());
+  // await page.route('https://cdn.jsdelivr.net/npm/monaco-editor@0.33.0/**', (route) => route.abort());
```
why did you comment this out?

ales-tsurko commented on ui/addressVerification/steps/AddressVerificationStepSignature.tsx:
```diff
@@ -142,7 +142,7 @@ const AddressVerificationStepSignature = ({ address, signingMessage, contractCre
   })();
 
-  const contactUsLink = <span>contact us <Link href="mailto:help@blockscout.com">help@blockscout.com</Link></span>;
+  const contactUsLink = <span>contact us <Link href="mailto:help@atlascan.com">help@atlascan.com</Link></span>;
```
please clarify the address in our common chat. I'm not sure we'll have like this one.

ales-tsurko commented on ui/pages/Home.pw.tsx:
```diff
@@ -48,20 +48,20 @@ test.describe('default view', () => {
     await expect(component).toHaveScreenshot({
-      mask: [ page.locator(configs.adsBannerSelector) ],
-      maskColor: configs.maskColor,
+      //mask: [ page.locator(configs.adsBannerSelector) ],
```
yeah, please check everywhere you commented out the ad banner test (if it is why I'm thinking of) and remove the code, instead of commenting it out. check my previous comment in similar place.

ales-tsurko commented on ui/shared/Web3ModalProvider.tsx:
```diff
@@ -37,7 +37,7 @@ const getConfig = () => {
       blockExplorers: {
         'default': {
-          name: 'Blockscout',
+          name: 'Atleta',
```
"Atlascan"?

ales-tsurko commented on ui/shared/texts/AdminSupportText.tsx:
```diff
@@ -9,7 +9,7 @@ const AdminSupportText = ({ className }: Props) => {
     <Box className={ className }>
       <span>Need help? Contact admin team at </span>
-      <Link href="mailto:help@blockscout.com">help@blockscout.com</Link>
+      <Link href="mailto:help@atlascan.com">help@atlascan.com</Link>
```
I've written about the address above. Please, verify

r8bywork committed:
Fix: Update variables, remove comments, and update Playwright screenshots

- Removed outdated comments
- Updated variables for clarity
- Updated Playwright screenshots to reflect changes in UI
- Update description for meta
- Delete deploy/services/logs/prod/error.log
